src/Bicep.Core/Semantics/FunctionOverloadBuilder.cs

```diff
@@ -93,6 +96,12 @@ public FunctionOverloadBuilder WithVariableParameter(string namePrefix, TypeSymb
             return this;
         }
 
+        public FunctionOverloadBuilder WithEvaluator(FunctionOverload.EvaluatorDelegate expressionEmitter)
```

Comment:
>expressionEmitter [](http://example.com/codeflow?start=88&length=17)

Should be renamed to match the new changes. (It's not using the emitter anymore.)

Reply:
done.

Reply:
done